x/cdp/keeper/draw.go

```diff
@@ -89,8 +89,10 @@ func (k Keeper) RepayPrincipal(ctx sdk.Context, owner sdk.AccAddress, denom stri
 		return err
 	}
 
+	totalDebt := cdp.Principal.Add(cdp.AccumulatedFees)
```

Comment:
Is `totalDebt` a good name?

Reply:
`totalPrincipal` is more inline with the naming conventions we are using.

I'm fine with this, would just add a comment for future maintainers:
 `Note: assumes  cdp.Principal and cdp.AccumulatedFees don't change during calculations`